app/cdap/components/AddTransformation/iconStore.tsx

```diff
@@ -25,3 +25,22 @@ export const SearchIcon = (
     />
   </svg>
 );
+
+export const TickIcon = (
```

Comment:
plz move TickIcon to separate file inside IconStore folder
next time onwards start moving icons into separate files inside this folder

thank you

Reply:
Done